mirdata/datasets/cuidado.py

```diff
@@ -0,0 +1,226 @@
+
+    **Beat and Bar Annotations:**
+
+    The beat annotations are structured as `.beats` files, where each line represents a beat with its timestamp and beat ID. For example, a line `9.430022675 3` indicates that the third beat of a bar is located at 9.43 seconds. This format is particularly useful for identifying downbeats, as they correspond to beats with ID = 1.
+
+    **Annotation Methodology:**
+
+    The dataset's annotations are based on the tempo guidelines of each cuidado dance style. Initial annotations were generated using a beat tracker, and then manually adjusted for accuracy. This method ensures that the annotations reflect the characteristic rhythms of each dance style.
+
+    **Applications:**
+
+    The Cuidado Rhythm Dataset is ideal for developing and testing algorithms for beat tracking, tempo estimation, and rhythm analysis in cuidado dance music. It can also be used for educational purposes, offering insights into the rhythmic structures of various ballroom dance styles.
+
+    **Acknowledgments and References:**
+
+    This dataset was created with the collaboration of experts in cuidado dance music. We extend our gratitude to those who contributed their knowledge and expertise to this project. For detailed information on the dataset and its creation, please refer to the associated research papers and documentation.
+    
+    [1] Gouyon F., A. Klapuri, S. Dixon, M. Alonso, G. Tzanetakis, C. Uhle, and P. Cano. An experimental comparison of audio tempo induction algorithms. Transactions on Audio, Speech and Language Processing 14(5), pp.1832-1844, 2006.
+
+    [2] Böck, S., and M. Schedl. Enhanced beat tracking with context-aware neural networks. In Proceedings of the International Conference on Digital Audio Effects (DAFX), 2010.
+    
+    [3] Dixon, S., F. Gouyon & G. Widmer. Towards Characterisation of Music via Rhythmic Patterns. In Proceedings of the 5th International Society for Music Information Retrieval Conference (ISMIR). 2004.
+"""
+
+import os
+import csv
+import logging
+import librosa
+import numpy as np
+from typing import BinaryIO, Optional, TextIO, Tuple, List
+
+from mirdata import annotations, core, download_utils, io, jams_utils
+
+
+BIBTEX = """
+@ARTICLE{1678001,
+    author={Gouyon, F. and Klapuri, A. and Dixon, S. and Alonso, M. and Tzanetakis, G. and Uhle, C. and Cano, P.},
+    journal={IEEE Transactions on Audio, Speech, and Language Processing}, 
+    title={An experimental comparison of audio tempo induction algorithms}, 
+    year={2006},
+    volume={14},
+    number={5},
+    pages={1832-1844},
+    doi={10.1109/TSA.2005.858509}}
+"""
+
+INDEXES = {
+    "default": "1.0",
+    "test": "1.0",
+    "1.0": core.Index(filename="cuidado_full_index_1.0.json"),
+}
+
+REMOTES = None
+
+LICENSE_INFO = (
+    "Creative Commons Attribution Non Commercial Share Alike 4.0 International."
+)
+
+DOWNLOAD_INFO = """
+    Unfortunately most of the Cuidado dataset is not available for download.
```

Comment:
What do you mean `most`? I think it would be cool to specify what is available and what is not.